frontend/app/forms/petition-for-dismissal-or-acquittal/petition-for-dismissal-or-acquittal.pdf.component.js

```diff
@@ -87,12 +87,16 @@ export default function PetitionForDismissalOrAcquittal_Pdf({
         </PositionedString>
         <PositionedString dataKey="case.caseNumber" left="54.1%" top="52.9%" />
         <PositionedString dataKey="case.judgeName" left="54.1%" top="57.5%" />
-        <PositionedString dataKey="case.caseNumber" left="17.4%" top="65.5%" />
```

Comment:
Good catch noticing that this in the wrong position. It looks like the case number still needs to be on the form in a different position, though? Should we add it back and move it to the correct position?

<img width="469" alt="image" src="https://user-images.githubusercontent.com/5524384/66276320-01b1ca00-e84f-11e9-8d3f-fc5c332fa044.png">

Reply:
Yes, that's what I thought but I don't know where the correct position is.

Reply:
> (a) The Following for case was field as a result of arrest:

@joeldenning It looks like that question is handled by `case.resultOfArrest` on line 102.

>Good catch noticing that this in the wrong position. It looks like the case number still needs to be on the form in a different position, though? Should we add it back and move it to the correct position?

I don't think it needs to be added back in. `case.CaseNumber` is already handled on line 88.

Reply:
I'm pretty sure this field should just be `case.caseNumber`. We don't need a `case.resultOfArrest` field at all since the result of arrests is the creation of a case.

Reply:
@tuckersamuelsen can you comment on this?